src/game/components/AvailbleActionsIntro.tsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 import { darken } from 'polished';
 import { useParams } from 'react-router-dom';
 import { Flex } from 'shared/components/Flex';
-import { Link } from 'shared/components/index';
+import { Link, SkipLink } from 'shared/components/index';
 import { SpeechBubbleWrapper, SpeechBubbleBorder } from 'shared/components/SpeechBubble';
```

Comment:
why not
import { Flex, Link, SkipLink, SpeechBubbleWrapper, SpeechBubbleBorder } from 'shared/components';